controller/rats_manager.py

```diff
@@ -0,0 +1,191 @@
+import time
+import traceback
+from datetime import datetime
+from typing import Optional
+
+from misc import daemon
+from submodules.model import enums
+from misc.notification import (
+    send_notification_created,
+)
+from submodules.model import (
+    RecordAttributeTokenStatistics,
+)
+from submodules.model.enums import AttributeState
+from submodules.model.business_objects import (
+    project,
+    attribute,
+    record,
+    tokenization,
+    general,
+    notification,
+)
+from handler.tokenizer_handler import get_tokenizer_by_project
+from misc.util import get_docs_from_db, send_websocket_update
+from submodules.model.models import RecordTokenizationTask
+
+
+def trigger_rats_creation(
+    project_id: str, user_id: str, attribute_id: Optional[str] = None
```

Comment:
the optional parameter attribute_id is never provided, `finalize_task` doesn't seem to differentiate between attribute specific and not